src/Models/IoTWorkspaceProject.ts

```diff
@@ -185,14 +185,38 @@ export class IoTWorkspaceProject extends IoTWorkbenchProjectBase {
         createTimeScaffoldType, openInNewWindow, OpenScenario.createNewProject);
   }
 
-  async openProject(
-      scaffoldType: ScaffoldType, openInNewWindow: boolean,
-      openScenario: OpenScenario): Promise<void> {
+  /**
+   * Validate workspace config file exists.
+   * Handles situation when workspace config file has a different name.
+   * @param scaffoldType
+   */
+  async validateWorkspaceConfigFilePath(scaffoldType: ScaffoldType):
+      Promise<void> {
     if (!await FileUtility.fileExists(
             scaffoldType, this.workspaceConfigFilePath)) {

```

Comment:
We may need to revisit the APIs in FileUtility, I don't think a general file utility API should care about the scaffold type.